charts/ontology/templates/service.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+{{ include "ontology.labels" . | indent 4 }}
+spec:
+  ports:
+    - name: http
+      port: 8080
+  selector:
```

```suggestion
  type: LoadBalancer
  selector:
```

This will enable the service to be accessible outside the cluster. For now this will randomnly generate a new ip address each time the service is created. I think we can worry about static ip and dns once we confirm the service is functioning